Directory.Packages.props

```diff
@@ -1,36 +1,59 @@
 <Project>
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-    <SystemVersion>9.0.3</SystemVersion>
     <System10Version>10.0.0-preview.2.25163.2</System10Version>
-    <MicrosoftExtensionsVersion>9.0.3</MicrosoftExtensionsVersion>
     <MicrosoftExtensionsAIVersion>9.3.0-preview.1.25161.3</MicrosoftExtensionsAIVersion>
   </PropertyGroup>
-  <ItemGroup>
-    <!-- Product dependencies -->
-    <PackageVersion Include="coverlet.collector" Version="6.0.4">
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-      <PrivateAssets>all</PrivateAssets>
-    </PackageVersion>
-    <PackageVersion Include="Microsoft.Bcl.Memory" Version="$(SystemVersion)" />
+
+  <!-- Product dependencies netstandard -->
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageVersion Include="Microsoft.Bcl.Memory" Version="9.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.AI.Abstractions" Version="$(MicrosoftExtensionsAIVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.Hosting.Abstractions" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="8.0.0" />
+    <PackageVersion Include="System.IO.Pipelines" Version="8.0.0" />
+    <PackageVersion Include="System.Net.ServerSentEvents" Version="$(System10Version)" />
+    <PackageVersion Include="System.Text.Json" Version="8.0.5" />
```

Comment:
Any particular reason you reference 8.0.0 for everything else, but 8.0.5 for this one? (it might add a small road bump for adoption in say, a large desktop product).

Reply:
<img width="674" alt="image" src="https://github.com/user-attachments/assets/0e54a64e-f035-402b-9090-49e0e9f057c1" />